runtime/test-runtime/src/lib.rs

```diff
@@ -551,9 +552,14 @@ sp_api::impl_runtime_apis! {
 		}
 	}
 
-	// Dummy implementation to continue supporting old parachains runtime temporarily.
-	impl p_v0::ParachainHost<Block> for Runtime {
-		fn validators() -> Vec<p_v0::ValidatorId> {
+	// REVIEW: I have _no idea_ what I'm really doing here, what invariants are necessary for the
```

Comment:
Note that there is a runtime_api_impl module in `runtime/parachains` that you should be able to forward calls to.

Reply:
You'll need to add all parachains modules to the test runtime for that, though.

Reply:
Do I? It looks like this is the only change that the runtime actually requires:

https://github.com/paritytech/polkadot/blob/73ded05a6359bd2c32c650817a84b82baf2ed55a/runtime/test-runtime/src/lib.rs#L472-L473

Will adding the rest of them help somehow?